docs/kit/operation-view/installation_guide.md

```diff
@@ -0,0 +1,116 @@
+   ```bash
+   helm repo add tractusx-edc https://eclipse-tractusx.github.io/charts/dev
+   helm install edc tractusx-edc/tractusx-connector --version 0.3.3 --set vault.hashicorp.enabled=true --set postgresql.enabled=true --set vault.hashicorp.url=<YOUR URL> --set vault.hashicorp.token=<YOUR TOKEN> --set daps.clientId=<YOUR ID> --set controlplane.endpoints.management.authKey=<YOUR AUTHKEY> --set postgresql.username=<YOUR USERNAME> --set postgresql.password=<YOUR PW> --set postgresql.jdbcUrl=<YOUR JDBC URL> --set backendService.httpProxyTokenReceiverUrl=<YOUR URL> --set daps.url=<YOUR URL>
+   ```
+5. Check if the pods are running:
+   ```bash
+   kubectl get pods
+   ```
+6. To see logs of the pods, run the following command:
+   ```bash
+   kubectl logs -f <pod-name>
+   ```
+7. Following endpoints are now available in default setup:
+   - controlplane: 
+     - default: port `8080` `/api` 
+     - management: `8081` `/management`
+     - control: port `8083` `/control` 
+     - protocol: port `8084` `/api/v1/ids`
+     - observability: port `8085` `/observability`
+     - metrics: port `9090` `/metrics`
+   - dataplane:
+     - default: port `8080` `/api`
+     - public: port `8081` `/api/public`
+     - control: port `8083` `/api/dataplane/control`
+     - observability: port `8085` `/observability`
+     - metrics: port `9090` `/metrics`
+    
+8. How to continue?  
+You could register your connector in the Catena-X portal, for more information see [here](https://portal.int.demo.catena-x.net/documentation/?path=docs%2F02.+Technical+Integration).
+
+FAQ
+------------
+### How should the DAPS service URL be specified?  
+The URL of the DAPS depends on the dataspace you want to connect to. The best thing here is to contact a contact person and ask for the DAPS URL of the dataspace you want to join. The example URL will connect you to the Catena-X Int-Stage DAPS.
+
+### How should the client ID of the connector be obtained?
+The client ID results from the SKI and AKI of the connector certificate. Fur further information on certificates of Catena-X, please take a look at the [Catena-X Connector Registration FAQ](https://portal.int.demo.catena-x.net/documentation/?path=docs%2F02.+Technical+Integration%2F01.+Connector+Registration%2F07.+FAQ.md). 
+
+### It seems like the connector specific properties are all optional?
+Not changing these values does not prevent the connector from starting, but describe the operator of the connector. It therefore makes sense to adapt these values individually to the intended use.
+
+### What exactly are the following values and where can they be obtained?
+- `controlplane.endpoints.authKey`: An authkey plays an important role in security and access control, helping to prevent unauthorized access and ensure that only trusted actors can access certain services. Please set this value individually to a confidential key. This value will be used later to be allowed to call some ControlPlane endpoints.
```

Comment:
better: "this value is used as access token to control plane endpoints". people are familiar with basic security concepts.

Is there any reason you are explaining this one in particular, and no others?